admin-dev/themes/new-theme/.webpack/common.js

```diff
@@ -50,7 +50,7 @@ module.exports = {
     cms_page_form: './js/pages/cms-page/form',
     contacts: './js/pages/contacts',
     currency: './js/pages/currency',
-    currency_form: './js/pages/currency/form',
+    currency_form: ['babel-polyfill', './js/pages/currency/form'],
```

Comment:
Must be added everywhere, not only here :thinking:

Reply:
OK, it wasn't easy for me but I ended up finding the solution thanks to `babel-plugin-transform-runtime` and with `"useBuiltIns": "usage"`